proto/viam/app/v1/app.proto

```diff
@@ -527,6 +527,7 @@ message Fragment {
   google.protobuf.Timestamp created_on = 6 [(tagger.v1.tags) = "bson:\"created_on\""];
   string organization_name = 7;
   bool readonly = 8;
+  int32 use_count = 9;
```

Comment:
Not sure if it will be used for this but we should avoid relying on use_count to signify if the frontend should be allowed to delete the fragment. Use a different field for that like `delete_allowed`.

Reply:
we want to add a column showing the number of robots using this fragment, which is what the number signifies. Delete allowed isn't visible, and currently just blocks the delete if you try to delete a fragment that is being used somewhere.